frontend/src/util/alphavantage_keys.js

```diff
@@ -0,0 +1,3 @@
+const apiKey = "7XF2MDOK9MZJBHFJ";
```

Comment:
This file should probably be added to gitignore

Reply:
lol yes rather urgently too 😱 good catch!